### PR TITLE
[ Packaging ] Meta Package for NNTrainer

### PR DESCRIPTION
**Pull request by jijoongmoon**
NNTrainer Meta :
 - nntrainer-core
 - nnstreamer-nntrainer if nnstreamer_filter is on
 - capi-nntrainer if with tizen

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: jijoong.moon <jijoong.moon@samsung.com>

### CODE REVIEW

**Review by zhoonit — APPROVED** [review]
LGTM

File: packaging/nntrainer.spec [hunk 1]
```diff
@@ -254,6 +268,8 @@ cp -r result %{buildroot}%{_datadir}/nntrainer/unittest/
 %postun -p /sbin/ldconfig
 
 %files
```
Comment (kparichay) [inline packaging/nntrainer.spec:270]: remove this? as nntrainer package depends on nntrainer-core, these files will be installed by core.

Comment (jijoongmoon) [reply packaging/nntrainer.spec:270]: This should be empty package. It's only for the resolve dependency.

Comment (kparichay) [reply packaging/nntrainer.spec:270]: I am not so sure about this.
Please once check - https://wiki.tizen.org/Security/Application_installation_and_Manifest. It says that "Each RPM package in MSM must have a manifest file".

**Review by kparichay — APPROVED** [review]

File: packaging/nntrainer.spec (hunk 1 above)
Comment (jijoongmoon) [reply packaging/nntrainer.spec:270]: I think this is not actually installed. The packages which has dependency with it and has manifest file are installed.

Comment (myungjoo) [reply packaging/nntrainer.spec:270]: If a package does NOT have a binary or executable (.so, .out, .sh, and so on), it has no execution incurred by the package, so MSM has nothing to register for the package. So, if the package has no "execution files", you don't need to care MSM. That's why we do not add manifest file to packages without .so, .out, .sh files.

**Pull request merged**